### PR TITLE
Upload sourcemaps to Sentry on publish

### PR DESCRIPTION
**Pull request by DMarby**
Tested here: https://github.com/Kong/insomnia/runs/7078908702?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/release-build.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ jobs:
         shell: bash
         run: npm run app-package
         env:
+          NODE_OPTIONS: '--max_old_space_size=6144'
```
Comment (DMarby) [inline .github/workflows/release-build.yml:44]: This is now needed as enabling sourcemaps cause Node to OOM in CI otherwise.

**Review by johnwchadwick — APPROVED** [review]
I worry a bit about the NODE_OPTIONS stuff, but otherwise everything looks good to me.

**Review by marckong — APPROVED** [review]
I reserve the view as @johnwchadwick 's. I wonder what would be path forward correcting it if needed.

File: .github/workflows/release-build.yml [hunk 2]
```diff
@@ -118,6 +119,15 @@ jobs:
...
             packages/insomnia-inso/artifacts/*
 
+      - name: Upload source assets for Sentry
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.os }}-sentry
+          path: |
+            packages/insomnia/build/*.js
+            packages/insomnia/build/*.map
+            !packages/insomnia/build/yarn-standalone.js
```
Comment (filfreire) [inline .github/workflows/release-build.yml:129]: What does the `!` character do here?

**Review by filfreire — APPROVED** [review]
LGTM! 👍

File: .github/workflows/release-build.yml (hunk 2 above)
Comment (DMarby) [reply .github/workflows/release-build.yml:129]: It negates the path so that yarn-standalone doesn't get copied even tho it's covered by the globs above

**Pull request merged**